src/components/SharedComponents/SubjectPhasePicker/SubjectPhasePicker.tsx

```diff
@@ -385,10 +536,36 @@ const SubjectPhasePicker: FC<SubjectPhasePickerData> = ({
     return title;
   };
 
+  const handleConfirmSubject = () => {
+    setShowPhases(true);
+    setTimeout(() => {
+      setIsMobileLotPickerModalOpen(false);
+    }, 50);
```

Comment:
Why do we need the `setTimeout` here?

Reply:
I had initially added this for a better transition between the subject to phase picker, but you're correct it's no longer needed